src/main/webapp/js/script.js

```diff
@@ -76,3 +77,93 @@ function showOrHideNotes(version) {
     $('div.setup-panel div a.btn-success').trigger('click');
 });
 
+
+//Fetch data from API endpoint
+ 
+// Fetch for landing page
+let doms = ["web", "android", "games", "ios"]
+
+async function getPlatforms() {
+    const response = await fetch('/v1/platforms');
+    const platforms = await response.json();
+  
+    count = 0;
+    for (let plat in platforms){
+       if (plat == doms[count]){
```

Comment:
confused about why you need count and are checking this. If you want to make sure that the platforms are what you expect them to be(ie plat is one of web, android, games, ios) you could just do if (doms.includes(plat))

Reply:
no that is to match the platform with the actual id in the HTML so that I know where to place it.